code/importers/cloc/setup.sh

```diff
@@ -1,4 +1,6 @@
 cd "$(dirname "$0")"
+command -v croc ||  echo "Install cloc and rerun"
```

Comment:
croc? did you mean cloc?

Reply:
@SRS-WRKS

Reply:
YES. I am sorry. cloc. It is is an embarrassing typo .